package.json

```diff
@@ -14,8 +14,7 @@
     "test-teamcity": "mocha test/functional test/programmatic --reporter lib/teamcity.js",
     "test-teamcity-example": "mocha test/example/exampleSimple.js --reporter lib/teamcity.js",
     "lint": "eslint lib/*.js test/**/*.js",
-    "lint-fix": "eslint --fix lib/*.js test/**/*.js",
-    "prepare": "npm run test"
```

Comment:
What's the reason behind dropping prepare? However I didn't realize it was in so many npm lifecycle events now.

probably prepack would be more appropriate for context it was being used to run the tests before being packaged and published to npm. I don't mind tidying that up once your PR is merged in.

Reply:
I removed prepare script because travis-ci failed to execute it. It tried to run test script and failed because no mocha is installed at the moment when prepare script is launched. My guess is: it happened because mocha is not part of dev dependency. Not sure if it should be installed because of peer dependency.

Reply:
proof: https://travis-ci.org/github/travisjeffery/mocha-teamcity-reporter/builds/766902127

Reply:
So do you expect anything else from my side?
I tried to put `npm install mocha & npm run test` into prepare script, but it did not work https://travis-ci.org/github/travisjeffery/mocha-teamcity-reporter/builds/766904317

Reply:
Nah, ill investigate that thanks for solving it in the meantime!